src/components/marsPhotoViewer/mars-viewer.scss

```diff
@@ -0,0 +1,4 @@
+
+.photoViewer-header { 
```

Comment:
can we rename this to `photo-viewer-header`? Using kebab case is standard practice for classes